runtime/shared_common/CompositeCache.cpp

```diff
@@ -1355,7 +1356,7 @@ SH_CompositeCacheImpl::startup(J9VMThread* currentThread, J9SharedClassPreinitCo
 				 *  - store information regarding -Xnolinenumbers
 				 */
 				if (isCacheInitComplete() == false) {
-					initializingNewCache = true;
+					_initializingNewCache = true;
```

Comment:
You can consider adding a trace point here.

Reply:
Already added an event.